ElementX/Sources/Other/SwiftUI/Views/PlaceholderAvatarImage.swift

```diff
@@ -23,16 +23,19 @@ struct PlaceholderAvatarImage: View {
     private let contentID: String?
     
     var body: some View {
-        ZStack {
-            bgColor
-            
-            // This text's frame doesn't look right when redacted
-            if redactionReasons != .placeholder {
-                Text(textForImage)
-                    .padding(4)
-                    .foregroundColor(.white)
-                    .font(.system(size: 200).weight(.semibold))
-                    .minimumScaleFactor(0.001)
+        GeometryReader { proxy in
```

Comment:
This is a personal preference here, feel free to ignore, but I've always found it makes the usage clearer as `geometry.size.width <= 30`.

```suggestion
        GeometryReader { geometry in
```

Reply:
Applied! ;)